mvp_example/src/main/java/com/distillery/android/blueprints/mvp/adapter/ToDoListDiffCallback.kt

```diff
@@ -0,0 +1,14 @@
+package com.distillery.android.blueprints.mvp.adapter
+
+import androidx.recyclerview.widget.DiffUtil
+import com.distillery.android.domain.models.ToDoModel
+
+class ToDoListDiffCallback : DiffUtil.ItemCallback<ToDoModel>() {
+    override fun areItemsTheSame(p0: ToDoModel, p1: ToDoModel): Boolean {
+        return p0 == p1
```

Comment:
The implementation of the callback isn't exactly correct. This method should check if variables point to a single instance:
```suggestion
        return p0 === p1
```

Reply:
I'm not sure, that it is a good way. I changed it, but it's connected with uniqueId(not memory id).
https://codelabs.developers.google.com/codelabs/kotlin-android-training-diffutil-databinding/#3

Reply:
yeah, sorry, you actually should unique ids, so it should be something like
```kotlin
todo1.id == todo2.id
```